src/InteractiveWindow/Editor/InteractiveWindow.cs

```diff
@@ -66,7 +62,6 @@ private enum ReplSpanKind
         private bool _adornmentToMinimize;
 
         private readonly IWpfTextView _textView;
-        private readonly History _history;
```

Comment:
I suspect that's where we'll end up, but I'm trying to break it down into logical pieces so the consequences are more obvious.